tutorials/FF/pipe-pipe/pimpleFoam-pimpleFoam/Fluid1/constant/transportProperties

```diff
@@ -12,6 +12,4 @@ transportModel  Newtonian;
 
 nu              nu [ 0 2 -1 0 0 0 0 ] 1e1;
 
-rho             rho [1 -3 0 0 0 0 0] 1e-0;
```

Comment:
(testing comment) rho is in nu

Reply:
So, in any case, we don't need it anywhere at the moment, right?